src/Microsoft.DocAsCode.Build.SchemaDriven/Processors/MarkdownInterpreter.cs

```diff
@@ -38,7 +39,21 @@ private static string MarkupCore(string content, IProcessContext context, string
             (context.FileLinkSources).Merge(mr.FileLinkSources);
             (context.UidLinkSources).Merge(mr.UidLinkSources);
             (context.Dependency).UnionWith(mr.Dependency);
-            return mr.Html;
+
+            HtmlDocument htmlDocument = new HtmlDocument();
+            htmlDocument.LoadHtml(mr.Html);
+            foreach (var node in htmlDocument.DocumentNode.Descendants())
+            {
+                if (!node.HasAttributes)
+                {
+                    continue;
+                }
+                //if the node have this attribute, it's markdown content and add new attribute
+                if (!node.GetAttributeValue("sourceStartLineNumber", false))
+                    node.SetAttributeValue("jsonPath", path);
```

Comment:
remove this later in `RemoveDebugInfo.cs`

Reply:
Already add 'jsonPath' into RemoveDebugInfo.cs